### ISSUE TITLE
Android: ScrollView not scrollable when TalkBack is on

### ISSUE THREAD

**Issue by sladek-jan** [software-mansion/react-native-gesture-handler#2472]
### Description

On a real Android device with TalkBack enabled, ScrollView from react-native-gesture-handler cannot be scrolled properly (only the alternative scrolling with 3 fingers works).
This can be easily reproduced with a freshly created React Native app.

### Steps to reproduce

1. make sure you have a real Android device ready
2. npx react-native@0.70.9 init scrollview_bug --version 0.70.9
3. cd scrollview_bug
4. replace the content of App.tsx with the following:
```
import React from 'react';
import {View} from 'react-native';
import {GestureHandlerRootView, ScrollView} from 'react-native-gesture-handler';

export default function App() {
  return (
    <GestureHandlerRootView>
      <ScrollView scrollEventThrottle={4}>
        <View style={{height: 600, backgroundColor: 'orange'}} />
        <View style={{height: 600, backgroundColor: 'red'}} />
        <View style={{height: 600, backgroundColor: 'blue'}} />
      </ScrollView>
    </GestureHandlerRootView>
  );
}
```

5. npx react-native run-android
6. while in the app, turn TalkBack on (on my Huawei device it's by pressing both volume buttons for 3 seconds)
7. try scrolling down with 2 fingers (this is standard for TalkBack , see [docs](https://support.google.com/accessibility/android/answer/6151827?hl=en#zippy=%2Cversion-up)). In my case it's very difficult or impossible.

**Observations:**
- scrolling with 3 fingers still works
- when I import ScrollView directly from react-native, it works fine

https://user-images.githubusercontent.com/4992232/236197816-2755bbac-9ea7-4be0-88da-572f9256157b.mov

### Snack or a link to a repository

- easy to reproduce with a freshly created RN-app and the code snippet provided above

### Gesture Handler version

2.9.0

### React Native version

0.70.9

### Platforms

Android

### JavaScript runtime

Hermes

### Workflow

React Native (without Expo)

### Architecture

Paper (Old Architecture)

### Build type

Debug mode

### Device

Real device

### Device model

Huawei P30 Lite (Android 12)

### Acknowledgements

Yes

**Comment by m-bert** [comment]
Hi @sladek-jan! I've just checked that and it seems to work fine. My guess is that [this PR](https://github.com/software-mansion/react-native-gesture-handler/pull/2234) fixed it. Could you please check that and confirm whether the issue is solved?
